common/Cargo.toml

```diff
@@ -31,25 +31,22 @@ sp-core = { workspace = true, features = ["full_crypto"] }
 sp-io = { workspace = true, features = ["disable_allocator", "disable_panic_handler", "disable_oom"] }
 sp-runtime = { workspace = true, features = ["serde"] }
 substrate-prometheus-endpoint = { workspace = true, optional = true }
-subxt = { workspace = true, optional = true }
 subxt-signer = { workspace = true, features = ["subxt", "sr25519"], optional = true }
 tangle-subxt = { workspace = true, optional = true }
 
 [target.'cfg(all(any(target_arch = "wasm32", target_arch = "wasm64"), target_os = "unknown"))'.dependencies]
 getrandom = { workspace = true, features = ["js"] }
 
 [features]
-default = ["substrate", "std", "tangle-testnet", "subxt/native"]
+default = ["substrate", "std", "tangle-testnet", "tangle-subxt/default"]
```

Comment:
Yeah, that was just to get it to build while I focused on other things. I accidentally put `subxt/native` in the `default` feature instead of `std`: https://github.com/webb-tools/tangle/blob/d8cecdfe6a735977885b4a01c504b29823d9ba94/tangle-subxt/Cargo.toml#L24-L29

Fixing that would require `tangle-subxt` 0.3.0 (I think). I'll fix it soon.